src/components/ViewPCOL/Details.tsx

```diff
@@ -106,7 +113,7 @@ const ViewPCOLDetails = () => {
             <Text id="Disclaimer">
               {pcol.data.Disclaimers?.map((item) => (
                 <>
-                  {item}
+                  {item?.slice(1)}
```

Comment:
Suggest adding a comment above this, or next to it, saying this is removing the prefix of g/p that is added to identify global vs program.